frame/bounties/src/weights.rs

```diff
@@ -39,7 +39,7 @@
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
-use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use frame_support::{traits::Get, weights::{ComputationWeight as Weight, constants::RocksDbWeight}};
```

Comment:
Shouldn't the `weights.rs` files be returning the full weight, not just computation weight? Where would we expect the PoV bandwidth weight dimension to come from?

Reply:
Yes, this is a needed follow up to regenerate weights with the proper template and values. This is just a backwards compat hack for now.